Servers/UI/OJS.Servers.Ui/ClientApp/src/components/submissions/details/SubmissionDetails.tsx

```diff
@@ -41,12 +47,18 @@ const SubmissionDetails = () => {
             return;
         }
 
-        const { problem: { id: problemId }, isOfficial } = currentSubmission;
+        const { problem: { id: problemId }, isOfficial, user: { id: userId } } = currentSubmission;
 
-        (async () => {
-            await getSubmissionResults(problemId, isOfficial);
-        })();
-    }, [ currentSubmission, getSubmissionResults ]);
+        if (!user.permissions.canAccessAdministration) {
```

Comment:
`canAccessAdministration` is not correctly named anymore.

Reply:
I created a const named isUserAdminOrLecturer that is set to the user having the access right to the administration.